[2021-07-22 13:34:12,069] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: psotgrressdftest.restart_test 2021-07-22T13:33:48.979471+00:00 [queued]>
[2021-07-22 13:34:12,285] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: psotgrressdftest.restart_test 2021-07-22T13:33:48.979471+00:00 [queued]>
[2021-07-22 13:34:12,294] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-07-22 13:34:12,300] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-07-22 13:34:12,302] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-07-22 13:34:12,354] {taskinstance.py:1087} INFO - Executing <Task(TriggerDagRunOperator): restart_test> on 2021-07-22T13:33:48.979471+00:00
[2021-07-22 13:34:12,381] {standard_task_runner.py:52} INFO - Started process 471 to run task
[2021-07-22 13:34:12,399] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'psotgrressdftest', 'restart_test', '2021-07-22T13:33:48.979471+00:00', '--job-id', '2039', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/POSTGRESRSERS.py', '--cfg-path', '/tmp/tmpzg4o11ko', '--error-file', '/tmp/tmpld3k3k6q']
[2021-07-22 13:34:12,401] {standard_task_runner.py:77} INFO - Job 2039: Subtask restart_test
[2021-07-22 13:34:12,673] {logging_mixin.py:104} INFO - Running <TaskInstance: psotgrressdftest.restart_test 2021-07-22T13:33:48.979471+00:00 [running]> on host 5795a622f846
[2021-07-22 13:34:13,114] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=psotgrressdftest
AIRFLOW_CTX_TASK_ID=restart_test
AIRFLOW_CTX_EXECUTION_DATE=2021-07-22T13:33:48.979471+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-07-22T13:33:44.061990+00:00
[2021-07-22 13:34:20,149] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=psotgrressdftest, task_id=restart_test, execution_date=20210722T133348, start_date=20210722T133412, end_date=20210722T133420
[2021-07-22 13:34:20,242] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-07-22 13:34:20,339] {local_task_job.py:151} INFO - Task exited with return code 0
